client/tests/e2e/specs/test_user_details_modal.ts

```diff
@@ -0,0 +1,35 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+describe('Check user details modal', () => {
+  beforeEach(() => {
+    cy.visitApp();
+    cy.login('Boby', 'P@ssw0rd.');
+    cy.get('.organization-card__manageBtn').click();
+    cy.get('.users-container').find('.user-list-item').eq(2).find('.options-button').invoke('show').click();
+    cy.get('.user-context-menu').find('.menu-list').find('ion-item').as('menuItems');
+    cy.get('@menuItems').eq(3).contains('View details').click();
+  });
+
+  afterEach(() => {
+    cy.dropTestbed();
+  });
+
+  it('Tests user details modal', () => {
+    cy.get('.user-details-modal').as('modal').find('ion-header').contains('User details');
+    cy.get('@modal').find('.ms-modal-content').as('modal-content').find('ion-text').eq(0).contains('Name');
+    cy.get('@modal-content').find('ion-text').eq(1).contains('Joined');
+    // cspell:disable-next-line
+    cy.get('@modal-content').find('ion-text').eq(2).contains('Jaheira');
+    cy.get('@modal-content').find('ion-text').eq(3).contains('one second ago');
+    cy.get('@modal-content').find('ion-text').eq(4).contains('Shared workspaces');
+    cy.get('@modal-content').find('ion-list').find('ion-card').as('workspace-cards').should('have.length', 2);
+    cy.get('@workspace-cards').eq(0).find('ion-text').contains('Workspace1');
+    cy.get('@workspace-cards').eq(0).find('ion-icon');
```

Comment:
Add a `.should('exist')`. I think Cypress does it implicitely but it's always better to state it explicitely.